Mathlib/Topology/ContinuousMap/Basic.lean

```diff
@@ -429,19 +429,15 @@ namespace Homeomorph
 variable {α β γ : Type*} [TopologicalSpace α] [TopologicalSpace β] [TopologicalSpace γ]
 variable (f : α ≃ₜ β) (g : β ≃ₜ γ)
 
+instance instContinuousMapClass : ContinuousMapClass (α ≃ₜ β) α β where
+  map_continuous f := f.continuous_toFun
+
 /-- The forward direction of a homeomorphism, as a bundled continuous map. -/
-@[simps]
-def toContinuousMap (e : α ≃ₜ β) : C(α, β) :=
+@[simps, deprecated _root_.toContinuousMap (since := "2024-10-12")]
```

Comment:
What's the replacement for the generated `simps` lemma?

Reply:
Fixed.